src/vivarium_csu_alzheimers/components/alzheimers.py

```diff
@@ -50,11 +51,11 @@ def setup(self, builder: Builder) -> None:
     def compute_transition_rate(self, index: pd.Index) -> pd.Series:
         entrance_time = self.population_view.get(index)["bbbm_entrance_time"]
         current_time = self.clock()
-        time_diff_numeric = (current_time - entrance_time).dt.total_seconds() / (
+        time_diff_numeric_years = (current_time - entrance_time).dt.total_seconds() / (
             365.0 * 24 * 3600
-        )  # years
+        )
 
-        return gamma_hazard(time_diff_numeric) * self.step_size
+        return hazard(time_diff_numeric_years, BBBM_HAZARD_DIST)
```

Comment:
It's a little lost in the `gamma_hazard` vs `hazard` change, but the critical bit is that I'm no longer `* self.step_size`